AnnoyancesFilter/sections/popups.txt

```diff
@@ -429,7 +429,6 @@ blokt.com##div[id^="coinzilla_expandable_widget"]
 cp.glico.jp###sec_modal_banner
 gd.ru#$#.paywall[data-window-name-for-ga] { display: none !important; }
 gd.ru#$#.megaShadow { display: none !important; }
-gd.ru#$#body[style^="overflow:"] { overflow: auto !important; }
```

Comment:
better do not ungroup CSS rules. In its current variant, it is more convenient.

Reply:
Please revert CSS changes an I will merge.

Reply:
@SKEIDs you can sort simple rules, but `#$#` is not necessary here, because we have some rules for each site that are easy to understand or change later on.

Reply:
thank you for advice!